routes/api.js

```diff
@@ -488,6 +499,7 @@ router.get('/create_account', async (req, res) => {
         res.status(400).json({ error: 'error_api_token_invalid' });
       }
     } catch (err) {
+      console.log(err);
```

Comment:
should we log this via the logging library?

Reply:
No sorry I've forgot to remove this line it was there for test purposes